bin/pre-commit-hook.js

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env node
 
+/**
```

Comment:
Hehe, yes I had that issue too when commented out linter part and staged file for commit by accident :)

Reply:
yeah, seems like we need to teach prettier about `#!` blocks.

Reply:
Apparently, prettier loves our credo and `never stop learning` part :)